.github/workflows/prices_check.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Get prices diff
         shell: bash
         run: |
-          git diff ${{ github.event.pull_request.base.sha}}..${{ github.event.pull_request.head.sha }} models/prices/prices_tokens.sql | grep '^\+ ' > scripts/new_lines.txt
+          git diff ${{ github.event.pull_request.base.sha}}..${{ github.event.pull_request.head.sha }} models/prices/**/*.sql | grep '^\+ ' > scripts/new_lines.txt
```

Comment:
I was thinking this line recursively scan all .sql files in the directory, and there is a pipe `|` operation after git diff, so in the `grep` step the diff from all files get merged into one flow/place, and write into a single file `new_lines.txt` by `>` operation.